djconnectwise/admin.py

```diff
@@ -170,11 +170,17 @@ class ProjectTypeAdmin(admin.ModelAdmin):
 
 
 @admin.register(models.ProjectPhase)
-class ProjectPhase(admin.ModelAdmin):
+class ProjectPhaseAdmin(admin.ModelAdmin):
     list_display = ('id', 'description', 'project')
     search_fields = ['description']
 
 
+@admin.register(models.ProjectTeamMember)
+class ProjectTeamMemberAdmin(admin.ModelAdmin):
+    list_display = ('id', 'member', 'project', 'start_date', 'end_date')
+    search_fields = ['member', 'project']
```

Comment:
This needs to be something more like `member__name` and `project__name`. Currently when you try to search it blows up with `Related Field got invalid lookup: icontains`. I would say allow searching on a few project fields, like name and id so related objects can be found in multiple ways. Same with member. Do what makes sense.

Reply:
Yep I missed that. Thanks for catching it!